examples/SingleDatePickerWrapper.jsx

```diff
@@ -65,6 +69,10 @@ const defaultProps = {
   displayFormat: () => moment.localeData().longDateFormat('L'),
   monthFormat: 'MMMM YYYY',
   phrases: SingleDatePickerPhrases,
+
```

Comment:
this line should be removed; properties in object literals aren't padded by blank lines.